docs/src/routes/docs/components/autocomplete-field.svx

```diff
@@ -0,0 +1,133 @@
+---
+name: AutocompleteField
+relatedComponents:
+  - name: Autocomplete
+    link: ./docs/components/autocomplete
+  - name: AutocompleteOption
+    link: ./docs/components/autocomplete-option
+---
+
+<script>
+  import { AutocompleteField } from 'attractions';
+  import { ChevronDownIcon } from 'svelte-feather-icons';
+  import Showcase from 'src/containers/docs/showcase.svelte';
+  import ColorPreview from 'src/components/docs/color-preview.svelte';
+  import RelatedComponents from 'src/components/docs/related-components.svelte';
+
+  function sleep(ms) {
+    return new Promise(resolve => setTimeout(resolve, ms));
+  }
+
+  async function* getOptions(text) {
+    await sleep(1000);
+    yield [
+      { name: text, details: 'Optional' },
+      { name: `it highlights the match: ${text}` },
+    ];
+  }
+
+  let selection = [];
+</script>
+
+
+# AutocompleteField
+
+A plain unstyled input element with a dropdown for suggestions.
+
+The [==`Autocomplete`==](./docs/components/autocomplete) component uses it to provide a full-featured autocomplete solution with chips for selection, it may be better if you don't intend to display the selection on your own.
+
+<RelatedComponents components={relatedComponents} />
+
+<Showcase>
+  <div slot="showcase">
+    <AutocompleteField {getOptions} bind:selection />
+    <pre>{JSON.stringify(selection, null, 2)}</pre>
+  </div>
+  <div slot="source">
+
+```svelte
+<script>
+  async function* getOptions(text) {
+    await sleep(1000);
+    yield [
+      { name: text, details: 'Optional' },
+      { name: `it highlights the match: ${text}` },
+    ];
+  }
+
+  let selection = [];
+</script>
+
+<AutocompleteField {getOptions} bind:selection />
+<pre>{JSON.stringify(selection, null, 2)}</pre>
+```
+
+  </div>
+</Showcase>
+
+
+## Properties {#properties}
+
+### Functional Properties
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| ==**`getOptions`**==&nbsp;<span class="required">*</span> | `async function*` | An async generator of suggestions. <br /> Receives the input from the text field and is expected to yield promises that resolve to arrays of objects `{ name, details }`. |
```

Comment:
> and is expected to yield promises

They don't necessarily have to be promises, direct values also work with `await` and `Promise.all`. A note like this can be useful for users who have the data already and think they need to wrap it in a `Promise`

Reply:
Yeah, but wrapping a value in a `Promise.resolve` isn't that difficult, besides, writing an async function will ensure that. I think it's ok